bin/lib/release.js

```diff
@@ -56,6 +57,10 @@ export default (releaseType) => {
         throw new Error('No .gitignore file');
       }
 
+      let outputFile = path.resolve(__dirname, `../../styles/${OUTPUT_CSS_FILENAME}`);
+      console.log(`- Deleting output file ${outputFile}`.green);
+      yield pUnlink(outputFile);
```

Comment:
Yes we get an error. Exists sync looks so horrible in that code base, but if it's fine for you...